File: py-many-resources/dummy.py
import pulumi


class Dummy(pulumi.ComponentResource):

    def __init__(self, name, deadweight, opts=None):
        super().__init__('examples:dummy:Dummy', name, {'deadweight': deadweight}, opts)

        self.deadweight = pulumi.Output.from_input(deadweight)
        self.register_outputs({'deadweight': self.deadweight})

File: misc/benchmarks/cs-many-resources/Dummy.cs
using Pulumi;

public class Dummy : ComponentResource
{
    [Output]
    public Output<string> Deadweight { get; private set; } = null!;

    public Dummy(string name, Input<string> deadweight, ComponentResourceOptions? options = null)
        : base("examples:dummy:Dummy", name, options)
    {
        this.Deadweight = deadweight;

        this.RegisterOutputs();
    }
}
